shared/src/main/scala/amf/core/AMFSerializer.scala

```diff
@@ -34,7 +34,7 @@ import org.yaml.model.YDocument
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class AMFSerializer(unit: BaseUnit, mediaType: String, vendor: String, env: RenderEnvironment) {
+class AMFSerializer(unit: BaseUnit, mediaType: String, vendor: String, env: RenderConfiguration) {
```

Comment:
Would make this take an AMFConfiguration for consistency with the AMFCompiler

Reply:
AMFCompiler is still receiving a compiler context.
Changing arguments for this clases is request at: https://www.mulesoft.org/jira/browse/APIMF-2983